recipes/machina/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "machina" %}
+{% set version = "1.1" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/raphael-group/{{ name }}/archive/v{{ version }}.zip
+  sha256: feb1efbedd32b8339f04699fade79fbc67bbd919df0a27f9a108f54c7531d208
+
+build:
+  number: 0
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - cmake>=3.9
+    - libboost 
+    - lemon
+    - wget
+    - zlib
```

Comment:
At least cmake and wget should get moved to the `build` section rather than `host`. Instead of libboost, can you specify boost-cpp? That will work better with project-wide pinnings.

Reply:
Made the change..`boost-cpp` forced us to make some changes to our `CMakelists.txt` as against `libboost`, but our package is better off for it!